test/mix/tasks/hellotest_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Mix.Tasks.HellotestTest do
+  use ExUnit.Case, async: true
+
+  setup do
+    :meck.new(Application, [:passthrough])
```

Comment:
I am sorry I missed to remove this block.
I tried to box the changes to the Application-variables, but this does not work using meck for single tests.
I removed it in the last commit.